### PR TITLE
Add help doc for debug tool

### PR DESCRIPTION
**Pull request by imay**
For #2018

### CODE REVIEW

File: docs/documentation/cn/developer-guide/debug-tool.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    #3 0x7fa5580fbbd4 in __libc_start_main (/opt/compiler/gcc-4.8.2/lib64/libc.so.6+0x21bd4)
+
+SUMMARY: AddressSanitizer: heap-buffer-overflow /home/ssd0/zc/palo/doris/core/be/src/olap/storage_engine.cpp:106 in doris::StorageEngine::open(doris::EngineOptions const&, doris::StorageEngine**)
+```
+
+从这段信息中该可以看到在`0x61900008bf80`这个地址我们尝试去写一个字节，但是这个地址是非法的。我们也可以看到 `[0x61900008bb80,0x61900008bf80)`这个地址的申请堆栈。
+
+**注意：开启这个选项是要影响程序的执行性能的，请慎重对线上的实例开启**
+
+**注意：如果开启了LSAN开关的话，tcmalloc就会被自动关闭**
```
Comment (morningman) [inline docs/documentation/cn/developer-guide/debug-tool.md]:
```suggestion
**注意：如果开启了ASAN开关的话，tcmalloc就会被自动关闭**
```

File: docs/documentation/cn/developer-guide/debug-tool.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+这样就能够生成一张BE执行的CPU消耗图。
+
+![CPU Pprof](../../../resources/images/cpu-pprof-demo.png)
+
+### perf + flamegragh
+
+这个是相当通用的一种CPU分析方式，相比于`pprof`，这中方式必须要求能够登陆到分析对象的物理机上。但是相比于pprof只能定时采点，perf是能够通过不同的事件来完成堆栈信息采集的。具体的的使用方式如下：
+
+```
+ perf record -g -p be_pid -- sleep 60
```
Comment (morningman) [inline docs/documentation/cn/developer-guide/debug-tool.md]:
```suggestion
perf record -g -p be_pid -- sleep 60
```

Comment (morningman) [reply docs/documentation/cn/developer-guide/debug-tool.md]: Is it `-- sleep 60` or `--sleep 60`(without space between `--` and `sleep`)

**Review by morningman — CHANGES_REQUESTED** [review]

File: docs/documentation/cn/developer-guide/debug-tool.md (hunk 2 above)
Comment (imay) [reply docs/documentation/cn/developer-guide/debug-tool.md]: It is `-- sleep 60`, which sleep is another shell command.

**Review by morningman — APPROVED** [review]
LGTM

**Comment by kangkaisen** [comment]
A great work!

**Pull request merged**